mantidimaging/core/operations/flat_fielding/flat_fielding.py

```diff
@@ -51,15 +49,10 @@ class FlatFieldFilter(BaseFilter):
     """Uses the flat (open beam) and dark images to normalise a stack of images (radiograms, projections),
     and to correct for a beam profile, scintillator imperfections and/or  detector inhomogeneities. This
     operation produces images of transmission values.
-
     In practice, several open beam and dark images are averaged in the flat-fielding process.
-
     Intended to be used on: Projections
-
     When: As one of the first pre-processing steps
-
     Caution: Make sure the correct stacks are selected for flat and dark.
-
     Caution: Check that the flat and dark images don't have any very bright pixels,
```

Comment:
Could you avoid removing the spaces here. They are needed for nice formatting in the docs.

https://mantidproject.github.io/mantidimaging/user_guide/operations/index.html#flat-fielding
With spaces
![image](https://github.com/mantidproject/mantidimaging/assets/74248560/270d746d-4034-490b-965a-154b6ca8fa5c)

Without spaces
![image](https://github.com/mantidproject/mantidimaging/assets/74248560/e75f7d78-a959-40fd-9c4d-7b81cc68ff63)